src/EventLoop.php

```diff
@@ -52,13 +69,17 @@ public function promiseRejected(\Throwable $throwable): Promise;
 
     /**
      * Creates a promise that will be resolved to null in a future tick of the event loop.
+     *
+     * @return Promise<void|null>
```

Comment:
Why void? :thinking: 
```suggestion
     * @return Promise<null>
```

Reply:
fixed